### PR TITLE
feat: add alien voiceover to a1g2

### LINKED ISSUE #84: 🚀 Feature: add alien voiceover for A1G2
for this game: image

Add an alien voiceover! Probably use the same voice as the other alien voice https://github.com/uclaacm/Playnet/pull/77/files 
[check out here for more voice options](https://talkrapp.com/speechSynthesis.html)

### Proposed Solution

* [ ] :wave: I may be able to implement this feature request

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by tr89on**
closes #84

### CODE REVIEW

**Comment by tr89on** [comment]
im not sure the voice fits for this part

File: src/components/Activities/Activity1/Game2/components/GameSlide.tsx [hunk 1]
```diff
@@ -49,19 +49,24 @@ function GameSlide(props: GameSlideProps): JSX.Element {
...
       playIncorrect();
+      setIncorrect(true);
       handleAlienState(ALIEN_STATE.ANGER);
     }
-    setIncorrect(newIncorrect);
   };
 
+  useEffect(() => {
+    const speech = new SpeechSynthesisUtterance(incorrect ? textIncorrect : textDefault);
+    speech.lang = 'de-DE';
```
Comment (reginawang99) [inline src/components/Activities/Activity1/Game2/components/GameSlide.tsx:64]:
```suggestion
    speech.lang = 'en-US';
```

**Review by reginawang99 — CHANGES_REQUESTED** [review]
one change then good to go!

**Review by reginawang99 — APPROVED** [review]
LGTM

**Pull request merged**